src/consumer_api_socket_endpoint.ts

```diff
@@ -127,14 +127,62 @@ export class ConsumerApiSocketEndpoint extends BaseSocketEndpoint {
         },
       );
 
-    const manualTaskReachedSubscription =
+    const boundaryEventTriggeredSubscription =
+      this.eventAggregator.subscribe(
+        Messages.EventAggregatorSettings.messagePaths.boundaryEventTriggered,
+        (boundaryEventTriggeredMessage: Messages.SystemEvents.BoundaryEventTriggeredMessage): void => {
+          socketIoInstance.emit(socketSettings.paths.boundaryEventTriggered, boundaryEventTriggeredMessage);
+        },
+      );
+
+    const intermediateThrowEventTriggeredSubscription: Subscription =
```

Comment:
Please make use of type-inference.

```suggestion
    const intermediateThrowEventTriggeredSubscription =
```